editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -1160,8 +1158,8 @@ void Polygon2DEditor::_uv_draw() {
 		uv_edit_draw->draw_circle(bone_paint_pos, bone_paint_radius->get_value() * EDSCALE, Color(1, 1, 1, 0.1));
 	}
 
-	rect.position -= uv_edit_draw->get_size();
-	rect.size += uv_edit_draw->get_size() * 2.0;
+	rect.position = Size2() - uv_edit_draw->get_size();
```

Comment:
`Size2` supports unary `-` so it should just be `rect.position = -uv_edit_draw->get_size();` no?

Reply:
OK. I wrote this so that I could, at first glance, associate the offset with the location. I will delete Size() and try again.
Thank you for your advice and response.